src/tsickle.ts

```diff
@@ -815,12 +821,17 @@ class Annotator extends ClosureRewriter {
     // only appear once and be merged.
     const suppressIdx = tags.findIndex(t => t.tagName === 'suppress');
     if (suppressIdx !== -1) {
-      const suppressions =
-          (tags[suppressIdx].text || '').replace(/^\{(.*)\}$/, '$1').split(',').map(s => s.trim());
-      if (suppressions.indexOf('checkTypes') === -1) {
-        suppressions.push('checkTypes');
+      const match = (tags[suppressIdx].text || '').match(/^\{(.*)\}(.*)$/);
+      if (!match) {
+        this.error(sf, `Failed to parse @suppress tag ${tags[suppressIdx].text}`);
```

Comment:
I wonder if this belongs in jsdoc.parse().  It already has some logic around parsing {foo} for types.